_maps/RandomRuins/SpaceRuins/TheDerelict.dmm

```diff
@@ -3372,7 +3372,7 @@
 "lq" = (
 /obj/structure/table,
 /obj/item/reagent_containers/glass/beaker{
-	list_reagents = list("sacid" = 50)
+	list_reagents = list(/datum/reagent/toxin/acid = 50)
```

Comment:
iirc, the maploader doesn't currently support generating instances from lists (which is why #38300 pops up every once and a while.) 

similar issue also caused #35518

Reply:
these are not instances, they are typepaths